src/pe/exception.rs

```diff
@@ -333,10 +333,45 @@ pub enum UnwindOperation {
     /// Save the lower 64 bits of a nonvolatile XMM register on the stack.
     SaveXMM(Register, StackFrameOffset),
 
-    /// Describes the function epilog.
+    /// Describes the function epilog location and size (Version 2).
     ///
-    /// This operation has been introduced with unwind info version 2 and is not implemented yet.
-    Epilog,
+    /// [UWOP_EPILOG] entries work together to describe where epilogues are located in the function,
+    /// but not what operations they perform. The actual operations performed during epilogue
+    /// execution are the reverse of the prolog operations.
+    ///
+    /// These entries always appear at the beginning of the [UnwindCode] array, with a minimum of
+    /// two entries required for alignment purposes, even when only one epilogue exists. The first
+    /// [UWOP_EPILOG] entry is special as it contains the size of the epilogue in its `offset_low_or_size`
+    /// field. If bit `0` of its `offset_high_or_flags` field is set, this first entry also describes
+    /// an epilogue located at the function's end, with `offset_low_or_size` serving dual purpose as
+    /// both size and offset. When bit `0` is not set, the first entry contains only the size, and
+    /// subsequent [UWOP_EPILOG] entries provide the epilogue locations.
+    ///
+    /// Each subsequent [UWOP_EPILOG] entry describes an additional epilogue location using a 12-bit
+    /// offset from the function's end address. The offset is formed by combining `offset_low_or_size`
+    /// (lower 8 bits) with the lower 4 bits of `offset_high_or_flags` (upper 4 bits). The epilogue's
+    /// starting address is calculated by subtracting this offset from the function's `EndAddress`.
+    Epilog {
+        /// For the first [UWOP_EPILOG] entry:
+        /// * Size of the epilogue in bytes
+        /// * If `offset_high_or_flags` bit 0 is set, also serves as offset
+        ///
+        /// For subsequent entries:
+        /// * Lower 8 bits of the offset from function end
+        offset_low_or_size: u8,
```

Comment:
unfortunately this is a breaking change so won't make it into the minor release. Also, if you anticipate in the future any enum variants/cases being added in the future, I strongly encourage to put a `#[non_exhaustive]` annotation on this `pub` enum so that we can safely add new cases in the future. If you don't anticipate it ever again having other variants, then we can leave it without